features/support/hooks.rb

```diff
@@ -1,52 +1,13 @@
-# encoding: utf-8
-#
-# Copyright (C) 2008-2013 NEC Corporation
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License, version 2, as
-# published by the Free Software Foundation.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License along
-# with this program; if not, write to the Free Software Foundation, Inc.,
-# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-#
-
-require 'trema/path'
-
-def wait_until_all_pid_files_are_deleted(timeout = 12)
-  elapsed = 0
-  loop do
-    fail 'Failed to clean up remaining processes.' if elapsed > timeout
-    break if Dir.glob(File.join(Trema.pid, '*.pid')).empty?
-    sleep 0.1
-    elapsed += 0.1
-  end
-  sleep 1
-end
-
 Before do
-  @aruba_timeout_seconds = 10
-  run 'trema killall'
-  wait_until_all_pid_files_are_deleted
+  @aruba_timeout_seconds = 5
 end
 
-Before('@slow_process') do
-  @aruba_io_wait_seconds = 1
+Before('@sudo') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.